angular5/app/app.component.ts

```diff
@@ -0,0 +1,101 @@
+import { Component, OnInit } from '@angular/core'
+
+import jss from 'jss'

```

Comment:
I would add jss-preset-default so that you can use camel case and other stuff

Reply:
It is imported, in the app.module.ts